### DISCUSSION TITLE
TaskbarX causing "show Desktop" button  to be not under cursor when mouse is in the very right bottom corner

### DISCUSSION THREAD

**Discussion by BlackRule** [valinet/ExplorerPatcher#254] [General]
I have muscule memory in Win10 of moving my mouse  in the very right bottom corner and clicking to do same as win+D shortcut . But with ExplorerPatcher's taskbar hovering the most right bottom pixel of the "show Desktop"  button doesn't trigger focus/hover on the button.
So in the meanwhile I have to use win+D instead of hovering the most right bottom pixel of my screen(
UPD: TaskbarX causes it. Your build of TaskbarX with start menu forced to follow the other buttons.

**Comment by valinet** [comment]
It’s not “ExplorerPatcher’s taskbar”, it’s Microsoft’s taskbar, largely the same one from Windows 10. This software merely enables that, and tries to provide a good experience around it.

As I said in a couple of posts already, please uninstall/disable any other software that might obviously or non-obviously interfere before complaining about something “EP does”. I understand what you describe, I tested that and it works as it always has on my setup: Windows 11, 22000.282, mostly stock, EP v32.2 (but it worked basically the same since v1). So, please double check for any software that might interfere and it should work just fine.

Thanks

**Reply by BlackRule** [reply]
Wow so fast reply +100500 respect. I was editing the msg and u replied with so big msg. UPD:  TaskbarX causes it. Your build of TaskbarX with start menu moving along with the othe buttons. IDK if the prev build has this issue cuz I want the startmenu to be in the center as well so want to use latest build.

**Reply by valinet** [reply]
> 
> 
> Wow so fast reply +100500 respect. I was editing the msg and u replied with so big msg. UPD: TaskbarX causes it. Your build of TaskbarX with start menu forced to follow the othe buttons.

=))) Thanks. Well, I have push notifications on my phone, so it vibrates for anything happening here. That said, it's both a blessing and a cursing, depends on how and when I look at it. 

Anyway, yeah, idk, it's not necessarly "my build of TaskbarX either", as I haven't done much in that besides adjusting a few things, haven't rewritten that or something like that, I don't mess with Show Desktop there, maybe TaskbarX in general does something there, idk. 

Also, I did that just for fun, just to see if it is doable, I am not running it anymore. At some point, in some timeline, I will hopefully get down to implementing that functionality proper in EP, but tbh, besides looks, it just is utterly useless for me, personally, as even on 4k screens, 10 windows in and there's nothing to center anymore with the buttons showing labels, plus, it is super annoying when closing a window and everything moves around on their own. It's pretty indeed to have icons centered, but it makes not the slightest sense from a productivity standpoint, it just gets in the way non-stop. And like, I don't care that much about looks, I just want the thing to work, I also have to use this computer for... y'know, actual work.
